meson.build

```
@@ -9,8 +9,13 @@ compiler = meson.get_compiler('d')
 if compiler.get_id() == 'llvm'
   d_extra_args = ['-vcolumns']
```

The ldc compiler also needs `-allinst` to be set, or it would throw same error as others (as far as I could test with Arch Linux's `ldc`)

```suggestion
  d_extra_args = ['-vcolumns', '-allinst']
```

---

It makes sense why we would add `-allinst` to all compilers given it's a frontend bug. I didn't add it for ldc2 since it was the only compiler that was working.

For the record, I tested with ldc2-1.39 and that one worked without the need of `-allinst`.